db/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+
+const getAllTopics = require('./controller.js');
+
+app.use(express.json());
+
+app.get('/api/topics', getAllTopics);
+
+app.use((err, req, res, next) => {
+    if(err.msg && err.status) {
+        res.status(err.status).send({msg: err.msg})
+    } else {
+    res.status(500).send({msg: 'internal server error'})
+    }
+});
```

Comment:
Nice start on error handling, although might be useful later are we currently using the custom error handler?